app/controllers/posts_controller.rb

```diff
@@ -8,4 +8,16 @@ def show
     @user = User.find_by_id(params[:user_id])
     @post = Post.find_by_id(params[:id])
   end
+
+  def create
+    @post = Post.new(author_id_id: params[:user_id], title: params[:post][:title],
+                     text: params[:post][:text], comments_counter: 0, likes_counter: 0)
```

Comment:
- [ OPTIONAL ] I highly recommend to add `post_params` mathod, since it is considered as [strong parameters](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides) so kindly check this link how to add strong parameters ✅✨🚀💯